docs/Tutorials/TLS.md

```diff
@@ -230,25 +149,114 @@ created.
 
 ### tlsclienttrust
 
-TLS trust mode for the client. The trust mode defines the servers to which the client connects. Options:
+TLS trust mode for the client. The trust mode defines the servers to which this node connects. Options:
 
 * `whitelist` - Nodes only connects to servers it has previously seen and have been added to `tlsknownservers`. 
 New servers are not added to `tlsknownservers`.
 
-* `tofu` - Trust-on-first-use. Node only connects same server for any given host. This is similar to how
+* `tofu` - Trust-on-first-use. Node only connects to the same server for any given host. This is similar to how
 OpenSSH works. 
 
 * `ca` -  Node only connects to servers with a valid certificate and chain of trust to one of the system 
 root certificates. Use the `SYSTEM_CERTIFICATE_PATH` environment variable to override the directory containing
  trusted root certificates.
 
+* `ca-or-whitelist` - Combination of `ca` and `whitelist`.
+
 * `ca-or-tofu` - Combination of `ca` and `tofu`. If a certificate is valid, it is always allowed and added 
 to the `tlsknownservers` list. If it is self-signed, it is allowed only if it is the first certificate 
 this node has seen for that host.
 
+* `insecure-record` - This node connects to any server, regardless of
+certificate, and is added to the `tlsknownservers` file.
+
+* `insecure-ca-or-record` - Combination of `ca` and `insecure-record`.
+
 * `insecure-no-validation` - Node connects to any server. Servers are added to the `tlsknownservers` file.
 
 ### tlsknownservers 
 
 TLS known servers for the client. The `tlsknownservers` contains the fingerprints of public keys of other
 nodes that this node has encountered for the `ca-or-tofu`, `tofu`, and `whitelist` trust modes.
+
+You can [use Orion to generate certificates](Generating-Certificates.md#generating-certificates-using-orion) and automatically populate the
+`tlsknownservers` file.
+
+## Client-to-Orion TLS Properties
+
+Properties to configure TLS between the client (for example Besu) and Orion.
+
+### clientconnectiontls
+
+TLS status options are:
+
+* `strict` - All connections to and from this node must use TLS with mutual
```

Comment:
Needs to be clearer that this is _all connections between Besu and Orion to and from this node must use TLS_

Reply:
Done